docs/versioned_docs/version-zenith/developer/typescript/506316-quickstart.mdx

```diff
@@ -0,0 +1,179 @@
+---
+slug: /developer/typescript/5063016/quickstart
+displayed_sidebar: "zenith"
+toc_max_heading_level: 2
+title: "Quickstart"
+---
+
+import PartialExperimentalDocs from '../../partials/_experimental.mdx';
+import PartialDeveloperTroubleshooting from '../../partials/_developer_troubleshooting.mdx';
+import PartialDeveloperQuickstartIntroduction from '../../partials/_developer_quickstart_introduction.mdx';
+import PartialDeveloperQuickstartTrivyTest from '../../partials/_developer_quickstart_trivy_test.mdx';
+import PartialDeveloperQuickstartConclusion from '../../partials/_developer_quickstart_conclusion.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Introduction
+
+<PartialDeveloperQuickstartIntroduction />
+
+## Requirements
+
+This quickstart assumes that:
+
+<!-- TODO(TomChv): How to add a link to the Typescript reference? -->
```

Comment:
Do we want to do this before we merge?

I don't know this off the top of my head, but maybe @helderco or @vikram-dagger do.